app/src/main/java/ch/blinkenlights/android/vanilla/ui/CustomElements/CustomTextView.java

```diff
@@ -0,0 +1,44 @@
+package ch.blinkenlights.android.vanilla.ui.CustomElements;
+
+import android.content.Context;
+import android.graphics.Rect;
+import android.util.AttributeSet;
+
+import androidx.appcompat.widget.AppCompatTextView;
```

Comment:
Why is this an AppCompatTextView? Is it using any feature not available in older api levels?

Reply:
My IDE suggested it.